freqtrade/commands/hyperopt_commands.py

```diff
@@ -97,18 +111,45 @@ def start_hyperopt_show(args: Dict[str, Any]) -> None:
                                      header_str="Epoch details")
 
 
-def _hyperopt_filter_trials(trials: List, only_best: bool, only_profitable: bool) -> List:
+def _hyperopt_filter_trials(trials: List, filteroptions: dict) -> List:
     """
     Filter our items from the list of hyperopt results
     """
-    if only_best:
+    if filteroptions['only_best']:
         trials = [x for x in trials if x['is_best']]
-    if only_profitable:
+    if filteroptions['only_profitable']:
         trials = [x for x in trials if x['results_metrics']['profit'] > 0]
 
+    if not filteroptions['only_best']:
```

Comment:
Why do you limit these options to be not compatible with '--best'?
In my opinion, this `if` can simply be removed, allowing `--best --min-avg-time NNN`

Reply:
agree